src/Dapr.Jobs/DaprJobsClient.cs

```diff
@@ -0,0 +1,106 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2024 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Net;
+using System.Net.Http.Json;
+
+namespace Dapr.Jobs;
+
+/// <summary>
+/// Defines client operations for managing Dapr jobs.
+/// </summary>
+public class DaprJobsClient
+{
+    private readonly HttpClient httpClient;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="DaprJobsClient"/> class.
+    /// </summary>
+    /// <param name="httpClient">The <see cref="HttpClient"/> used to communicate with the Dapr sidecar.</param>
+    public DaprJobsClient(HttpClient httpClient)
```

Comment:
We should be using gRPC to talk to the sidecar, when possible.  This brings up the question of how best to share creation of the underlying gRPC client, probably by internally exposing/sharing parts of the `DaprClientBuilder`.

Reply:
Yeah, after writing this version, I was digging into the gRPC prototypes and found that what's presented there is quite different from the HTTP API (which is why I deleted the comment asking for your urgent review). I'm working on prioritizing support for the gRPC side in lieu of HTTP (much as we do with Crypto).